include/vrv/options.h

```diff
@@ -557,6 +557,7 @@ class Options {
     OptionDbl m_defaultTopMargin;
     //
     OptionDbl m_bottomMarginHarm;
+    OptionDbl m_bottomMarginHeader;
```

Comment:
Rename to `m_bottomMarginPgHead`

Reply:
Done. Also I've moved usage of this margin from systems to more appropriate place page alignment.